framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -5741,9 +5746,66 @@ VkResult VulkanReplayConsumerBase::OverrideCreateRayTracingPipelinesKHR(
                                                             out_pPipelines);
     }
 
+    if ((in_deferredOperation != VK_NULL_HANDLE) &&
+        ((result == VK_OPERATION_DEFERRED_KHR) || (result == VK_OPERATION_NOT_DEFERRED_KHR)))
+    {
+        // Before running vkDeferredOperationJoinKHR, CreateRayTracingPipelinesKHR won't be processed. It means if any
+        // parameters are be modified before running vkDeferredOperationJoinKHR, it will fail. So processing deferred
+        // operation immediately after CreateRayTracingPipelinesKHR will be safer.
+        ProcessDeferredOperation(device, in_deferredOperation);
+    }
     return result;
 }
 
+void VulkanReplayConsumerBase::ProcessDeferredOperation(VkDevice device, VkDeferredOperationKHR deferred_operation)
+{
+    GFXRECON_LOG_ERROR_ONCE("The replayed application used VkDeferredOperationKHR. It might fail for rendering "
+                            "incorrect or VK_ERROR_DEVICE_LOST. The reason might be the device doesn't support "
+                            "rayTracingPipelineShaderGroupHandleCaptureReplay feature.");
```

Comment:
Add error info for that the device doesn't support rayTracingPipelineShaderGroupHandleCaptureReplay if replaying use VkDeferredOperationKHR.